### PR TITLE
instantiating players

### PR DESCRIPTION
**Pull request by RebaDel**
I think this should work

### CODE REVIEW

**Review by delm76 — CHANGES_REQUESTED** [review]
Go ahead and make the changes, and I will try the merge again.

File: mexicanTrainScoring.js [hunk 1]
```diff
@@ -9,13 +9,20 @@
 //# Accept number as input and set as variable.
 var numPlayers = prompt("How many players?", "Enter number here.");
 
+const players = [];
+
 if (Number.isInteger(numPlayers) === true){
-while (numPlayers > 0) {
 //instantiate player objects should include string name, int total, array roundScore
+  for (i = 0; numPlayers > 0; i++) {
+  var players[i] = {
```
Comment (RebaDel) [inline mexicanTrainScoring.js]: should not have var, all ';' in the object should be ','

Comment (delm76) [reply mexicanTrainScoring.js]: ok

**Comment by delm76** [comment]
Trying to get PR to merge.

**Review by delm76 — APPROVED** [review]
a

**Comment by delm76** [comment]
I found that there were conflicts that I had to resolve before committing and merging.  Conflicts must be resolved before PR can be merged.

**Review by delm76 — APPROVED** [review]

**Review by delm76 — APPROVED** [review]

**Pull request merged**